google_drive/google_docs.py

```diff
@@ -40,57 +49,95 @@ class GoogleDocumentBlock(XBlock):
 
     embed_code = String(
         display_name="Embed Code",
-        help="Google provides an embed code for Drive documents with a variety of settings. From inside a Google Drive document, select Publish to the Web from within the File menuto get to your Embed Code with your desired settings.",
+        help=(
+            "Google provides an embed code for Drive documents. In the Google Drive document, "
+            "from the File menu, select Publish to the Web. Modify settings as needed, click "
+            "Publish, and copy the embed code into this field."
+        ),
         scope=Scope.settings,
-        default=textwrap.dedent("""
-            <iframe
-                src="https://docs.google.com/presentation/d/1B22AsMwG7jgE39vm1BuiJClir_JUz1q077cPEKJR2mI/embed?start=false&loop=false&delayms=3000"
-                frameborder="0"
-                width="960"
-                height="569"
-                allowfullscreen="true"
-                mozallowfullscreen="true"
-                webkitallowfullscreen="true">
-            </iframe>
-        """))
+        default=DEFAULT_EMBED_CODE
+    )
 
-    def student_view(self, context={}):
+    alt_text = String(
+        display_name="Alternative Text",
+        help=(
+            "In situations where image is not available to the reader, the alternative "
```

Comment:
Alternative text describes an image and appears if the image is unavailable. Screen readers for the visually impaired also use alternative text.

(Or we could just go with the first sentence - the second might be overkill.)